A32-xds-circuit-breaking.md

```diff
@@ -80,13 +86,21 @@ fail the call with status UNAVAILABLE. Such failed calls will not be retried,
 but they will be recorded to the `total_dropped_requests` counts in 
 cluster-level load reports and reported to the load reporting server.
 
+Note an EDS LB policy may be responsible for handling `service_name` switch
+used for endpoint discovery, requests sent to both the old and new services
+should be aggregated to the same counter and the total is restricted by the
+configured limit. Even though the failed requests will be recorded and reported
+in separate stats message.
+
 ## Implementation
 
 ### Java
-- Maintain a counter in the CDS LB policy for counting the number of 
+- Add a `max_requests` field in the EDS LB config. The value will be taken
+from the cluster watch interface pushed by the XdsClient.
+- Maintain a counter in the EDS LB policy for counting the number of 
 outstanding requests.
-- In CDS LB policy, wrap the `SubchannelPicker` propagated from its child 
-EDS policy with the logic of checking the counter and make pick decision.
+- Wrap the `SubchannelPicker` propagated from each of EDS policy's child LB 
```

Comment:
The EDS policy is already wrapping the pickers from its children to handle the drops requested in EDS, so no new wrapping is needed.  We just need to modify the existing wrapper to also enforce the new max_requests field.

Reply:
The actual implementation is still flexible. One could choose to wrap another layer for this feature or could choose to add the logic to the existing wrapper. It is just for pointing out where to add the logic.

I updated the description a bit to make this more neutral.